src/components/Hero.astro

```diff
@@ -15,7 +15,7 @@ const signUpUrl =
   "https://gdg.community.dev/events/details/google-gdg-pescara-presents-devfest-pescara-2023/cohost-gdg-pescara";
 ---
 
-<section id="signUp" class="hero min-h-screen section-white">
+<section id="signUp" class="bg-white hero min-h-screen section-white">
```

Comment:
Questa è una modifica che inficia particolarmente sull'impatto della home. I frammenti neri erano in continuità col contenuto sottostante, aggiungendo questo bianco invece quella sensazione si perde - la sezione adesso è separata di netto, in un certo senso stona. Sicuri sia necessario?